storage/rocksdb/ha_rocksdb.cc

```diff
@@ -200,7 +193,7 @@ struct Rdb_open_tables_map {
   void init_hash(void) {
     (void)my_hash_init(&m_hash, my_core::system_charset_info, TABLE_HASH_SIZE,
                        0, 0, (my_hash_get_key)Rdb_open_tables_map::get_hash_key,
-                       0, 0);
+                       0, 0, 0); // TODO: instrument for PFS
```

Comment:
There is now, and probably more as other things need keys created for pfs instrumentation
https://jira.percona.com/browse/MYR-89